typings/types.d.ts

```diff
@@ -1184,6 +1259,21 @@ declare abstract class SequencerSoundManager {
 
 }
 
+declare abstract class SequencerCrosshair {
+
+  show(crosshair?: CrosshairData, callbacks?: CrosshairCallbackData): Promise<void>;
+
+  /**
+   * Collect overlapping placeable objects of a crosshair of a given placeable type
+   */
+  collect(
+    crosshair?: CrosshairData,
+    types?: String | Array<string>,
+    filterMethod?: Function
```

Comment:
The same typing hints I gave above apply here.

BTW if you want I'd be happy to make it so that your `types.d.ts` automatically generate from your TSDoc so you don't have to worry about keeping them synced up.